modules/ignition/resources/services/eta.service

```diff
@@ -0,0 +1,18 @@
+[Unit]
+Description=Ephemeral Token Agent
+After=docker.service
+Requires=docker.service
+
+[Service]
+TimeoutStartSec=0
+
+ExecStartPre=/usr/bin/docker pull ${eta_image}:${eta_tag}
+ExecStart=/usr/bin/docker run --name eta -p 8887:8887 --env-file /etc/eta/eta.env ${eta_image}:${eta_tag}
```

Comment:
I would expect the port here to be linked to the env file so it doesn't get changed in one place but not the other by mistake.

Reply:
Done